src/popup/pages/DonateError.vue

```diff
@@ -65,9 +66,10 @@ export default {
   methods: {
     async donate() {
       const { openDefaultModal } = useModals();
+      const api = useAeApiActions();
```

Comment:
I suggest to treat this a a regular composable, so pick the methods from returned object instead of creating the api constant.